ui/src/components/Rules/Rule/RuleCreator/RuleInput/index.tsx

```diff
@@ -4,31 +4,19 @@ import { IRule } from '../'
 import ConstantsContext, {
   IProperty,
   MediaType,
+  RulePossibility,
   RulePossibilityTranslations,
 } from '../../../../../contexts/constants-context'
 import { EPlexDataType } from '../../../../../utils/PlexDataType-enum'
 import _, { first } from 'lodash'
 
-enum RulePossibility {
```

Comment:
Found there were 2 duplicate enums; opting to delete this one and go with a single source of truth, given we already import from the other file

Reply:
Something on my todo list is to add a shared package for these duplicated types as well as API schemas (though there currently isn't any, another todo). I refactored to use Turborepo recently to make this possible. I've referenced @maintainerr/server from the UI in one place as an example, but this breaks pretty badly if that included file imports some other stuff and I think the separate package will fix that.

Reply:
yeah, having a shared schema types package would be really nice. there's a few other places I saw that are just using open strings etc and assuming things will serialize/deserialize correctly, but they totally could just go out of sync very easily